tests/characters/test_alice_can_grant_and_revoke.py

```diff
@@ -26,6 +26,7 @@
 
 from nucypher.characters.lawful import Bob
 from nucypher.config.characters import AliceConfiguration
+from nucypher.crypto.api import keccak_digest
```

Comment:
Btw it appears that Ethereum will switch from keccak to sha256 :scream: 
(not a 1st April joke)